### PR TITLE
feat(dashboard): add previews on templates

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

default templates
=> show preview image file on new tab

existing dashboards
=> open dahsboard detail page on new tab

### CODE REVIEW

File: src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue [hunk 1]
```diff
@@ -68,7 +68,7 @@
                             <p-text-pagination
                                 :this-page="existingTemplateState.thisPage"
                                 :all-page="existingTemplateState.allPage"
-                                @pageChange="(page) => handleChangePagination(page, 'EXISTING')"
+                                @pageChange="handleChangePagination($event, 'EXISTING')"
```
Comment (minseolee) [inline src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue:71]: These codes are not directly related with this PR 😅 🙇

File: src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue [hunk 2]
```diff
@@ -101,21 +106,42 @@ const defaultTemplateState = reactive({
...
+        }],
     })).slice(10 * (defaultTemplateState.thisPage - 1), 10 * defaultTemplateState.thisPage - 1)),
 });
 const existingTemplateState = reactive({
-    dashboards: computed<DashboardConfig[]>(() => [...store.state.dashboard.domainItems, ...store.state.dashboard.projectItems].filter((d) => d.name.includes(existingTemplateState.searchValue))),
+    dashboards: computed<DashboardModel[]>(() => [...store.state.dashboard.domainItems, ...store.state.dashboard.projectItems].filter((d) => d.name.includes(existingTemplateState.searchValue))),
     thisPage: 1,
     allPage: computed<number>(() => Math.ceil(existingTemplateState.dashboards.length / 10) || 1),
-    boardSets: computed<DashboardTemplateBoardSet[]>(() => existingTemplateState.dashboards.map((d: DashboardConfig) => ({
+    boardSets: computed<DashboardTemplateBoardSet[]>(() => existingTemplateState.dashboards.map((d: DomainDashboardModel & ProjectDashboardModel) => ({
```
Comment (WANZARGEN) [inline src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue:125]: suggestion: There is DashboardModel type that is already declared.

```suggestion
    boardSets: computed<DashboardTemplateBoardSet[]>(() => existingTemplateState.dashboards.map((d: DashboardModel) => ({
```

**Review by WANZARGEN — APPROVED** [review]
LGTM overal :)
I left a minor suggestion :)

File: src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue (hunk 2 above)
Comment (minseolee) [reply src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue:125]: `DashboardModel` is `DomainDashboardModel | ProjectDashboardModel`.
But in this case, I need `DomainDashboardModel & ProjectDashboardModel`. 
Because, the `existingTemplateState.dashboards` is consisted by both of them
